projects/igniteui-angular/src/lib/combo/combo-item.component.ts

```diff
@@ -99,6 +99,15 @@ export class IgxComboItemComponent extends IgxDropDownItemComponent implements D
         this.comboAPI.set_selected_item(this.itemID, event);
     }
 
+    /**
+     * @hidden
+     * @internal
+     */
+    handleFocus(event: MouseEvent) {
+        event.preventDefault();
+        this.dropDown.element.focus();
```

Comment:
Fairly certain you don't need this as the combo drop down is not focusable anyway. The prevent above does the heavy lifting